hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/validation/AttributeValidatorImplTest.java

```diff
@@ -103,7 +103,7 @@ void allowsFutureExpiryBeforeMaxLifetime() {
     void rejectsAnyNonFutureExpiry() {
         final var now = 1_234_567L;
         given(context.consensusNow()).willReturn(Instant.ofEpochSecond(now));
-        assertThatThrownBy(() -> subject.validateExpiry(now))
+        assertThatThrownBy(() -> subject.validateExpiry(now - 1))
```

Comment:
Since the expiry change was moved, would this still pass with `now`?

Reply:
Yes，changed it back to the original.